Helper/PathHelper.php

```diff
@@ -12,10 +12,14 @@ class PathHelper
     /**
      * @param \Symfony\Component\DependencyInjection\ContainerInterface $container
      */
-    public function __construct(ContainerInterface $container)
+    public function __construct(ContainerInterface $container, $symfony_version)
     {
-        $this->assetsHelper = $container->get('templating.helper.assets');
-        $this->routerHelper = $container->get('templating.helper.router');
+        if ($container->has('templating.helper.assets') && version_compare($symfony_version, '3.0.0') === -1) {
```

Comment:
Why would you want to check the Symfony version when checking if the service `templating.helper.assets`  exists is already enough?

```php
$this->assetsHelper = $container->has('templating.helper.assets')
    ? $container->get('templating.helper.assets')
    : $container->get('assets.packages');
```